packages/atomic/patches/@stencil+react-output-target+0.2.0.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/node_modules/@stencil/react-output-target/react-component-lib/createComponent.tsx b/node_modules/@stencil/react-output-target/react-component-lib/createComponent.tsx
```

Comment:
Cool approach to work-around the issue. It would be nice to open a PR for the stencil package too,